src/components/Header/index.tsx

```diff
@@ -308,6 +312,11 @@ function Header() {
         </HeaderSubRow>
 
         <Flex maxHeight={'22px'} justifyContent={'end'}>
+          {account && (
+            <HeaderButton onClick={toggleExpeditionsPopup} style={{ marginRight: '7px' }}>
+              &#128640;&nbsp;Expeditions
```

Comment:
what the hell are these chars? Seems worthy adding them to a variable on the side.
ok... I googled and then understood them but I don't think anyone will understand them just by looking at the code.